### ISSUE TITLE
Update Python on Lambda

### ISSUE THREAD

**Issue by ryscher** [CDL-Dryad/dryad-product-roadmap#2771]
We are contacting you as we have identified that your AWS Account currently has one or more Lambda functions using the Python 3.7 runtime.

We are ending support for Python 3.7 in AWS Lambda. This follows Python 3.7 End-Of-Life (EOL) reached on June 27, 2023 [1].

As described in the Lambda runtime support policy [2], end of support for language runtimes in Lambda happens in two stages. Starting November 27, 2023, Lambda will no longer apply security patches and other updates to the Python 3.7 runtime used by Lambda functions, and functions using Python 3.7 will no longer be eligible for technical support. In addition, you will no longer be able to create new Lambda functions using the Python 3.7 runtime. Starting January 25, 2024, you will no longer be able to update existing functions using the Python 3.7 runtime.

We recommend that you upgrade your existing Python 3.7 functions to Python 3.11 before November 27, 2023.

**Comment by sfisher** [comment]
I've been trying to install a newer version of Python on cloudshell that I use to create the lambda layer.  I've installed 3 times now but am having trouble getting it to use the 3.11.6 version in the shell.  It keeps reverting to 3.7 .  I'm trying again with pyenv for the 3rd time. `pyenv install -v 3.11.6`.  I thought it worked but 🤷 .  The last install just failed with missing openssl library which I installed with yum and am trying again, but I may cut out soon for the day and pick this up again when I get back after Thanksgiving.

The imports we have for these two layers are frictionless: imports json, time pprint, frictionless, urllib, xml, requests, sys and Excel to csv: imports json, time, pandas, requests, pathlib, datetime, re, urllib

I hope that will give what we need to install with pip3.11 .  Then I need to zip them up and upload to S3 as a layer.  Maybe I can give it a new lambda and name and test on our dev machine first.
